resource_estimator/src/system.rs

```diff
@@ -94,7 +94,7 @@ fn estimate_single<L: Overhead + LayoutReportData + PartitioningOverhead + Seria
             distillation_unit_templates,
             job_params.constraints().max_distillation_rounds,
         ),
-        logical_resources,
+        logical_resources.clone(),
```

Comment:
maybe a comments that the clone happens only once would be helpful

Reply:
Thank you, added a comment.